config/navigation.txt

```diff
@@ -48,10 +48,7 @@ Learn
       [/learn/develop/hardware/usb]
       [/learn/develop/hardware/i2c-and-spi]
     [/learn/develop/integrations]
```

Comment:
If you wish to execute my suggestion to remove this page, then you can remove this page from here as well..

Reply:
Fixed in fd5f464.